python/feathub/processors/local/local_processor.py

```diff
@@ -178,26 +211,41 @@ def materialize_features(
                 timestamp_field=features.timestamp_field,
                 timestamp_format=features.timestamp_format,
             )
-        elif isinstance(sink, FileSystemSink):
+        elif isinstance(sink, FileSystemSink) and _is_local_file_or_dir(sink.path):
```

Comment:
Let's update the docs of the FileSystem connector as the local processor supports HDFS now.